src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -124,6 +124,14 @@ The compatibility risk is low. RFC 2818 has deprecated this for nearly two decad
 [Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=308330)
 
 
+## VTTRegion-related bits of TextTrack
+
+The interface elements regions, addRegion() and removeRegion() have been removed from the WebVTT spec and will be removed from Chrome.
```

Comment:
I don't know your policy when it comes to unshipped features (which this is.) This should maybe also mention that VTTCue.region was added and that the VTTRegion interface was updated to better match the current spec draft. (I did some edits to that effect on the chromestatus entry.)